api/apps/api/src/modules/projects/projects-crud.service.ts

```diff
@@ -224,6 +225,9 @@ export class ProjectsCrudService extends AppBaseService<
           projectId: model.id,
           planningAreaGeometryId: createModel.planningAreaId,
         }),
+        this.commandBus.execute(
+          new SetProjectBlm(model.id, createModel.planningUnitAreakm2),
+        ),
```

Comment:
Does not seems right, please see:

https://vizzuality.atlassian.net/browse/MARXAN-995